test/sanity/code-smell/use-compat-six.py

```diff
@@ -20,9 +20,9 @@ def main():
         if path.startswith('test/sanity/pylint/plugins/'):
             continue
 
-        with open(path, 'r') as path_fd:
+        with open(path, 'rb') as path_fd:
             for line, text in enumerate(path_fd.readlines()):
-                match = re.search(r'((^\s*import\s+six\b)|(^\s*from\s+six\b))', text)
+                match = re.search(br'((^\s*import\s+six\b)|(^\s*from\s+six\b))', text)
```

Comment:
Instead of changing the regex to a byte regex, I think it's better to convert the lines into text.  The current code won't fail because it never touches the string that's been matched.  However, it will quickly produce mangled output on Pyhton3 if we were to use the matched text in and update to this code.  However, in some cases, that does have a knock on effect (making a text version of path to use in the print().  Changing the print format string to u"%s:%d:%d")

So this depends on whether mattclay is doing unicode sandwich for ansible-test code or not.